Api/Fluent Validation.md

```diff
@@ -126,10 +119,10 @@ public async Task CreateExternalLoginDetails(LoginDetailsDto googleUserInfo)
     }
     else
     {
-        foreach(var failure in results.Errors)
+        foreach(var error in results.Errors)
         {
-            Console.WriteLine("Property" + failure.PropertyName + "failed validation.");
-            Console.WriteLine("Error was: " + failure.ErrorMessage);
+            Console.WriteLine("Property" + error.PropertyName + "failed validation.");
+            Console.WriteLine("Error was: " + error.ErrorMessage);
```

Comment:
One important feature of integrating FluentValidation with MVC is to automatically set the `ModelState` and fill it with errors if the model is not valid. This enables us to use the native support for displaying error messages on the input forms. The example I've shown you in the Smart Parking solution has a base Controller with a generic method that does exactly that, so please take a look and add something similar (or the same) here as well.